src/xref2/tools.ml

```diff
@@ -206,12 +219,16 @@ type class_type_lookup_result =
 
 exception Type_lookup_failure of Env.t * Cpath.Resolved.type_
 
-exception Module_lookup_failure of Env.t * Cpath.Resolved.module_
-
 exception ModuleType_lookup_failure of Env.t * Cpath.Resolved.module_type
 
 exception ClassType_lookup_failure of Env.t * Cpath.Resolved.class_type
 
+type module_lookup_error = [
+  | `Failure of Env.t * Ident.module_ (** Found local path *)
```

Comment:
Let's make this less generic than `Failure` - though that is at least an improvement on `MyFailure` :-)

Reply:
This error case is when a `` `Local _ `` path is encountered, I'll call it `` `Local `` then